config/test.exs

```diff
@@ -0,0 +1,6 @@
+use Mix.Config
+
+config :recaptcha,
+  http_client: Recaptcha.Http.MockClient,
+  secret: "test_secret",
```

Comment:
So the only values you get are a public and private key. not the response itself. I did some playing around at it seems that even if you have the public and private key that "always pass a recaptcha" a different response is generated each time. I'm using the private key from that page in some of the tests.